doc/features/metrics/app-metrics/README.md

```diff
@@ -0,0 +1,140 @@
+# DataStax Metrics Provider based on App.Metrics
+
+This documentation refers to our provider based on App.Metrics. The code examples shown here only apply if this provider is used.
+
+## Installation
+
+This provider is available on the [CassandraCSharpDriver.AppMetrics nuget package].
```

Comment:
We could publish a `1.0.0-alpha1` ahead of time to reserve the name, otherwise doc generation might fail.

Reply:
Hmm good idea